http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/Boot.scala

```diff
@@ -128,21 +135,17 @@ object Boot extends IOApp {
         appDependencies.dockerDAO,
         googleDependencies.googleStorageService,
         googleDependencies.googleComputeService,
-        appDependencies.publisherQueue
-      )
-      val diskService = new DiskServiceInterp[IO](
-        ConfigReader.appConfig.persistentDisk,
-        appDependencies.authProvider,
-        appDependencies.serviceAccountProvider,
-        appDependencies.publisherQueue
+        appDependencies.publisherQueue,
+        diskService
```

Comment:
I added `diskService` as a dependency for `RuntimeService` in order to reuse the code to lookup and verify access to a source disk. Let me know if there is a better way to do this with Leo.